test/sim/items/ejectpack.js

```diff
@@ -35,4 +35,55 @@ describe('Eject Pack', function () {
 		battle.makeChoices();
 		assert.equal(battle.p2.requestState, 'switch');
 	});
+
+	it.skip("should not trigger until after all entrance abilities have resolved during simultaneous switches", function () {
+		battle = common.createBattle({gameType: 'doubles'});
+		battle.setPlayer('p1', {team: [
+			{species: 'Hydreigon', ability: 'intimidate', moves: ['sleeptalk']},
+			{species: 'Wynaut', moves: ['sleeptalk']},
+		]});
+		battle.setPlayer('p2', {team: [
+			{species: 'Morelull', ability: 'drought', item: 'ejectpack', moves: ['sleeptalk']},
+			{species: 'Mew', level: 1, ability: 'electricsurge', moves: ['sleeptalk']},
+			{species: 'Wynaut', moves: ['sleeptalk']},
+			{species: 'Wynaut', moves: ['sleeptalk']},
+		]});
+		battle.makeChoices();
+		assert.ok(battle.field.isWeather('sunnyday'));
+		assert.ok(battle.field.isTerrain('electricterrain'));
```

Comment:
```suggestion
		assert(battle.field.isWeather('sunnyday'));
		assert(battle.field.isTerrain('electricterrain'));
```

wtb banish `assert.ok` from the codebase